lib/src/test/java/com/thenewboston/utils/Mocks.kt

```diff
@@ -163,19 +163,23 @@ object Mocks {
         )
     }
 
-    fun bankTrustResponse(trust: Double = Some.trust) = BankTrustResponse(
+    fun bankTrustResponse(trust: Double = Some.trust) = Bank(
```

Comment:
We should now call this function `bank` instead of `bankTrustResponse` as well. If there's already another such function, we should consolidate their uses.